docs/01-app/03-building-your-application/02-data-fetching/03-server-actions-and-mutations.mdx

```diff
@@ -80,6 +80,8 @@ export function Button() {
 }
 ```
 
+> **Good to know**: The React documentation, considers the `create` function to be Server Function. [Learn more here](/app/getting-started/updating-data#server-functions).
```

Comment:
I think we would instead replace the paragraph before the code snippet and say "server function", with a link